### ISSUE TITLE
Some logo pictures in dataset

### ISSUE THREAD

**Issue by invisprints** [XiaoxiaoGuo/fashion-iq#6]
I notice there are some pictures that are not clothing, are they normal?
For example, ID B00APYBNR0 is not a dress, but in `cap.dress.train.json` it says:
![B00APYBNR0](http://g-ecx.images-amazon.com/images/G/01/x-locale/brands/small-logo/3034061011._CB359598500_SX150_.jpg)
```json
    {
        "target": "B00APYBNR0",
        "candidate": "B00A890DQY",
        "captions": [
            "is a short sleeved floral dress",
            "is a company name and not a dress"
        ]
    },
```
It  seems the first caption is wrong?

**Comment by zxh19976666** [comment]
How to divide the datasets into training set and test set，I have download all images，but how to divide it？

**Comment by invisprints** [comment]
Have yo seen the `image_splits` folder? The json files in it split the dataset.

**Comment by zxh19976666** [comment]
are you Chinese？

**Comment by zxh19976666** [comment]
Are you Chinese sir？

**Comment by zxh19976666** [comment]
> Have yo seen the `image_splits` folder? The json files in it split the dataset.

sir when I run this code it show No such file or directory: 'data/captions/dict.dress.json'  where is dict？

**Comment by VisualJoyce** [comment]
@zxh19976666 They are in the folder https://github.com/XiaoxiaoGuo/fashion-iq/tree/master/captions

**Comment by nayhtet72042** [comment]
Recall metric percentage is low, to increase accuracy and adjust what Can it be changed and used? Please reply sir.

**Comment by zxh19976666** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by nayhtet72042** [comment]
Thank you for replying, Sir.  I am waiting for your answer.

**Comment by nayhtet72042** [comment]
Recall metric percentage is low, to increase accuracy and adjust what Can it be changed and used? Please reply sir.
